tests/test_diffusers.py

```diff
@@ -5663,7 +5663,7 @@ def test_stable_diffusion_xl_inpaint_euler_lcm(self):
 
         expected_slice = np.array([0.6611, 0.5569, 0.5531, 0.5471, 0.5918, 0.6393, 0.5074, 0.5468, 0.5185])
 
-        assert np.abs(image_slice.flatten() - expected_slice).max() < 1e-2
+        assert np.abs(image_slice.flatten() - expected_slice).max() < 1e-1
```

Comment:
@yuanwu2017 why did you increase the tolerance level?

Reply:
the tests are passing on main as is

Reply:
Because the FP32 is used in the previous testcases which are copied from the diffusers[https://github.com/huggingface/diffusers/tree/main/tests/pipelines/stable_diffusion_xl], after applying this patch, AutoCast will be opened and caused the accuracy decrease.